### ISSUE TITLE
Cross-link to the github.io page from repo?

### ISSUE THREAD

**Issue by edmundhenley** [EquitableLettersSP/ELSP#28]
TLDR: suggestion for minor tweak: for increased discoverability of https://equitableletterssp.github.io/ELSP, might be worth putting visible links to it from:
* https://github.com/EquitableLettersSP
* https://github.com/EquitableLettersSP/ELSP

(e.g. like git-lfs has in About section: https://github.com/git-lfs/git-lfs ; alternatively/as well in README.md?)

Hi all,

_Quick intro: great project (heard via @aburrell's post to [MIST](https://www.mist.ac.uk/) mailing list)! Think the tooltips on the sample letters really nifty - great way of illustrating issues - v thought provoking!_

Context for suggestion above: am currently on my personal computer, and was wanting to look at your website (so I could promote it on social media!). 
As I don't have access to work email (and so mailing list post) in this context, I googled "Equitable letters space physics".
While the repo was listed at top of search results, the github.io page wasn't on 1st two pages of results.
No luck either searching with `site:github.io equitable letters` unfortunately - I've had to go back to work context to retrieve the actual link, https://equitableletterssp.github.io/ELSP .

Search results will be searcher-dependent of course, and results might well improve over time. But above might be a ~easy fix by repo maintainers to increase discoverability of the github.io page - at very least for those encountering repo first?
And might this improve early-stage [SEO](https://en.wikipedia.org/wiki/Search_engine_optimization) results for the github.io page too, having crosslinks from elsewhere (e.g. repo)?

**Comment by aburrell** [comment]
I think this is a really good point!  We should:

- [ ] Add links to the webpage in the README for this repo @melissaroses 
- [ ] Add a link on the project page @aburrell 
- [ ] Add a link on the CEDAR wiki @mjonesjr 
- [ ] Ask MIST if they will link us as a resource @johncoxon

**Comment by johncoxon** [comment]
I have added a permanent link to [this page on the MIST webpage](https://www.mist.ac.uk/community/conduct-and-support) and have also plugged ELSP with [a post on the news page](https://www.mist.ac.uk/news/274-equitable-letters-in-space-physics-elsp).

**Comment by edmundhenley** [comment]
Re your CEDAR/MIST points & publicity, am sure you've already thought of this, maybe even done/doing one - unsolicited mention just on offchance though - have you considered an article in [EOS](https://eos.org)?

Ask as I was very glad when I found [the EOS article on how to write a solid peer review](https://eos.org/science-updates/a-quick-guide-to-writing-a-solid-peer-review) the first time I had to do one. As the authors say, there's "suprisingly little training [...] on how to develop this essential skill" - none in my case! Made this article invaluable.

Reading your guide, it struck me that you're addressing a similarly weirdly-missing gap in guidance on a crucial aspect of our space science (and beyond!) "infrastructure", and that there might be a wide audience for this.

An article in EOS/similar may give good exposure, and perhaps there's a multiplier benefit of having multiple such resources in same place, especially if EOS curated these? Would make for an easy/lazy link target for organisations wanting aspects like this to be considered by their stakeholders, and who think resources like yours meet their needs - and are better than they could put together themselves (if I recall, I think I found the EOS article as was linked as an AGU journal's "how to do a review for us" instructions!)

**Comment by aburrell** [comment]
@edmundhenley this is a good idea.  We have planned to write for EOS or a similar news outlet after a year of running, to provide some statistics, give kudos to our reviewers, and try and convince people who don't immediately see the need for a resource like ELSP.  But your idea would reach people immediately, and we could even reference it in the one-year follow on!

**Comment by aburrell** [comment]
Ok, I started an outline for the EOS/A&G "tips for writing a good recommendation letter" and created an ELSP git repository for it so we can interact with it in different ways :)

**Comment by aburrell** [comment]
Closing this issue as the links have all been added now, and the letter article is underway.  Thank you for your input @edmundhenley!
